tests/aggregation_regress/tests.py

```diff
@@ -1950,7 +1954,10 @@ def test_non_nullable_fk_not_promoted(self):
 
 
 class SelfReferentialFKTests(TestCase):
-    def test_ticket_24748(self):
+    def test_self_referential_fk_aggregate(self):
```

Comment:
yeah sure since we touched it let's relocate it somewhere that makes more sense. 

I've seen a bunch of cases like this, but I didn't have the courage for bigger diffs... let me know if I should swing at that?